<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>342</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>选择要解析的doc文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fileBtn">
       <property name="text">
        <string>选择doc文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_FilePath"/>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="xml_title_info">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>281</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_xml_header">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>解析所有标题信息</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>70</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ExcuteResult">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>120</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>生成结果excel</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>40</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>excel文件标题</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ResultName"/>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>.xls</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget3">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>370</y>
      <width>441</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,1">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SizeConstraint::SetMaximumSize</enum>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>一级标题选择</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="selectFirstTitle"/>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_first_title"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1,1">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>二级标题选择</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="selectSecondTitle"/>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_second_title"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4" stretch=""/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_auto_produce">
       <property name="text">
        <string>自动生成三级以上表格</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
